pkg/services/oracle/neofs/neofs.go

```diff
@@ -50,7 +47,7 @@ func Get(ctx context.Context, priv *keys.PrivateKey, u *url.URL, addr string) ([
 		return nil, err
 	}
 
-	c, err := client.New(&priv.PrivateKey, client.WithAddress(addr))
+	c, err := client.New(client.WithDefaultPrivateKey(&priv.PrivateKey), client.WithURIAddress(addr, nil))
```

Comment:
Does it make us compatible with `grpcs://`?

Reply:
yes, this option can work with `grpcs` scheme, can work without scheme at all(previous behavior) and can configure TLS connection with the second argument if it is necessary(and if a scheme is `grpcs`. a scheme is primary)

one way should be chosen: `WithURIAddress` **OR** `WithAddress`[ + `WithTLS`]

Reply:
OK, so it'll make `grpcs://` support transparent for us, because we don't have any additional options to pass there and standard TLS will work just fine via this URI? The main question for me is can I specify `grpcs://url` in the configuration file and expect it to work with this code?

Reply:
> standard TLS will work just fine via this URI?

yes, it works fine with `nil` second arg(it is `*tls.Config`). configuring TLS connections is optional; passing URI in "grpcs://..." form is enough for server-side TLS encryption(server-side certificate checking)

> can I specify grpcs://url in the configuration file and expect it to work with this code?

yes, it will try to establish TLS encrypted connection then. without scheme(`"host:port"` format), it will work like only "WithAddress" option was passed